### PR TITLE
fix: enter response when image block selected

### PR DESCRIPTION
**Pull request by DiamondThree**

### CODE REVIEW

**Review by lawvs — APPROVED** [review]
Maybe need some new tests.

File: tests/embed.spec.ts [hunk 1]
```diff
@@ -65,3 +69,29 @@ test('can click and delete image', async ({ page }) => {
...
+  await initImageState(page);
+  await assertRichImage(page, 1);
+
+  await activeEmbed(page);
+  await page.keyboard.press('Enter', { delay: 50 });
+  await page.keyboard.type('aa');
+  await assertRichTexts(page, ['\n', 'aa']);
+});
+
+test('hover image block will show image-option', async ({ page }) => {
```
Comment (doodlewind) [inline tests/embed.spec.ts:86]: Please refer to https://github.com/toeverything/blocksuite/pull/310#discussion_r1049442419 about styling on the grouping of test cade code. Tight continuous spaghetti codes are not preferred.

File: tests/utils/asserts.ts [hunk 2]
```diff
@@ -101,6 +101,13 @@ export async function assertImageSize(
   expect(actual?.height).toEqual(height);
 }
 
+export async function assertImageOption(page: Page) {
+  const actual = await page
+    .locator('.affine-embed-editing-state-container')
+    .count();
```
Comment (lawvs) [inline tests/utils/asserts.ts:107]: Maybe should use [await expect(locator).toBeVisible()](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible)

File: tests/utils/asserts.ts [hunk 3]
```diff
@@ -101,6 +101,13 @@ export async function assertImageSize(
   expect(actual?.height).toEqual(height);
 }
 
+export async function assertImageOption(page: Page) {
```
Comment (lawvs) [inline tests/utils/asserts.ts:104]: It doesn't seem to need a separate function

File: tests/utils/actions/click.ts [hunk 4]
```diff
@@ -51,3 +51,7 @@ export async function switchShapeType(page: Page, shapeType: string) {
 export async function activeEmbed(page: Page) {
   await page.click('.resizable-img');
 }
+
+export async function focusCaption(page: Page) {
+  await page.click('.embed-editing-state>format-bar-button:nth-child(1)');
+}
```
Comment (lawvs) [inline tests/utils/actions/click.ts:57]: focus caption is not a common function, so should not include in the `utils/actions`

**Pull request merged**